changelog/11676.breaking.rst

```diff
@@ -0,0 +1 @@
+The classes :class:`~_pytest.nodes.Node`, :class:`~pytest.Collector`, :class:`~pytest.Item`, :class:`~pytest.File`, :class:`~_pytest.nodes.FSCollector` are now marked abstract (see :mod:`abc`).
```

Comment:
Thanks!

I added the last sentence, but I omitted the "Benefits" part since it seems a bit too much. I think those who are interested can check out the issue for the details.

Reply:
Sounds good, thanks!